integration-testing/config-local-host.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+set -e
+
+# install njs
```

Comment:
Help people figure out how to run the integration tests. Maybe add a section to the README? or a separate .md file